src/progress-indicator/progress-indicator.component.ts

```diff
@@ -59,7 +60,7 @@ export class ProgressIndicator {
 	}
 
 	@Input() steps = [];
-
+	@Input() vertical = false;
```

Comment:
I'm not sure if `@Input() type: "horizontal" | "vertical" = "horizontal";` or `@Input() direction: "horizontal" | "vertical" = "horizontal";` would make more sense ... I'm good with `@Input() vertical = false` though so I don't think it _needs_ to change.

Reply:
I just kept it 1:1 with the accepted PR's style shown here: https://github.com/IBM/carbon-components/pull/1717/files#diff-6e73227357afc271c1386a942097df26R61. If we were to depart from that here, the original component should be updated to use the same variable assignment style.

Reply:
We do deviate from the vanilla implementation where it makes sense from an API perspective - ex. in dropdown `@Input() type: "single" | "multi" = "single"` ... That seems to be the exception rather then the rule though, since we have a bunch of components with modifiers like this now, so it's going to be more consistent this way

Reply:
@cal-smith By being more consistent, do you mean we should continue the exception and use your suggested change, or avoid deviations and not commit your suggested change?

Reply:
Sorry that was ambiguous, I mean avoid deviations.

I'll leave this open if anyone else has a strong opinion, but I'm good to give this an approval 👍

Reply:
This _should_ be `type`. It goes very well with how we use `type` in combobox, overflow-menu-option, dropdown, dropdown-list...

Reply:
@zvonimirfras but is that a frequent exception as @cal-smith said, or is that the design direction/rule across the board going forward? Totally open to it myself, just wanna be sure I'm committing based on where we're going, not where we are.

Reply:
We took some time to think it over ... how about `orientation: "horizontal" | "vertical" = "horizontal"` ... we're trying to establish some primitives we can use across the codebase, and `orientation` makes a lot of sense :+1:

Reply:
Sorry for the delay!

Reply:
no prob, how's this? 1df8fce5c7f2b96f4ccbbefc60642dafc2ed33be

Reply:
Perfect 👍